prow/pluginhelp/pluginhelp.go

```diff
@@ -41,6 +43,14 @@ type PluginHelp struct {
 	// Events is a slice containing the events that are handled by the plugin.
 	// NOTE: Plugins do not need to populate this. Hook populates it on their behalf.
 	Events []string
+	// Commands maps a command name to a struct of its properties.
+	Commands map[string]struct {
+	  // Frequency suggests how often a command is used. Value ranges from 0 to 2, the SMALLER the
```

Comment:
You've got some formatting issues, will need to run `hack/update-gofmt.sh`

Reply:
thanks, that helps